### PR TITLE
fixed error in roc.py causing blank plots

### LINKED ISSUE #231: Blank plots in matplotlib guide
Blank plots showing up in `Comparing several models with one plot` and `Grid Plots` sections [here](https://sklearn-evaluation.ploomber.io/en/latest/reference/matplotlib.html)

### PR DESCRIPTION
**Pull request by gtduncan**
## Describe your changes
Found error in roc.py in which a call to plot() was missing a parameter. ROC curves should now be able to be graphed on subplots instead of separately from a blank plot, seen [here](https://sklearn-evaluation.ploomber.io/en/latest/reference/matplotlib.html) 

## Issue ticket number and link
Closes #231 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [x] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://sklearn-evaluation--250.org.readthedocs.build/en/250/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - [API Change] A new figure and axes is created (via `plt.subplots()`) when calling a plotting method with `ax=None`. Previously, the current axes was used (via `plt.gca()`) ([#211](https://github.com/ploomber/sklearn-evaluation/pull/211))
 - [Fix] Validating input elbow curve model has "score" method [#146] 
 - [Fix] Adds class labels for multi class roc plot (#209)
+- [Fix] Fixed error in which ROC curves would not graph on subplots
```
Comment (idomic) [inline CHANGELOG.md:9]: This should be [Docs], the text should probably be, Fixed ROC curves plots to show properly.

**Review by idomic — CHANGES_REQUESTED** [review]
Besides the changelog entry looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (idomic) [reply CHANGELOG.md]: Yeah, maybe we can call it a fix after all. @gtduncan

Comment (idomic) [reply CHANGELOG.md]: So if it fits both categories, we should probably take the code first approach.

Comment (idomic) [reply CHANGELOG.md]: Something weird is happening here as well, the tests are running for 3 hours already.

Comment (gtduncan) [reply CHANGELOG.md]: That's what I was about to ask about, but it seems to be happening to all the CI checks

Comment (gtduncan) [reply CHANGELOG.md]: I ran all the tests locally and they passed

Comment (idomic) [reply CHANGELOG.md]: Stay tuned, work on other issues :)

Comment (edublancas) [reply CHANGELOG.md]: the error has been fixed, please rebase

**Comment by idomic** [comment]
Nice job @gtduncan !

**Pull request merged**